src/Microsoft.DotNet.Interactive.CSharp.Tests/Microsoft.DotNet.Interactive.CSharp.Tests.csproj

```diff
@@ -39,4 +39,8 @@
     </PackageReference>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="Microsoft.NET.Test.Sdk" Version="16.9.4" />
```

Comment:
Remove all references to `Microsoft.NET.Test.Sdk` because Arcade handles this for us and it'll cause problems when Arcade isn't disabled.  The proper place to upgrade this package is [here](https://github.com/dotnet/interactive/blob/06ac1da7d95da4306e81624ddea927e64ffb98f4/Directory.Build.targets#L11).  N.b., however, that the version of Arcade we're using specifies a Test.SDK of 16.6.1, but since this only applies if DisableArcade=1, you can use the 16.9.4 if you like.

Reply:
doing it